tests/__snapshots__/data-loader.csr.spec.tsx.snap

```diff
@@ -35,7 +35,7 @@ Object {
     },
     Object {
       "state": Object {
-        "fc7e58063ef49874aea8e2210fc7a6a2f9836f52": Object {
```

Comment:
This is basically the issue, this hash `should` be the same on client and server, and it is for all browsers except IE11. This new library is far simpler, smaller and hopefully faster.

Reply:
Will be interested to see if this fixes some of the random issues we've had in IE11 between the SSR and client side render.